test/e2e/auth/audit.go

```diff
@@ -648,33 +662,44 @@ type auditEvent struct {
 }
 
 // Search the audit log for the expected audit lines.
-func expectAuditLines(f *framework.Framework, expected []auditEvent) {
+func expectAuditLines(f *framework.Framework, expected []auditEvent) (bool, error) {
```

Comment:
should be called `searchAuditLines` or something like that. Expect implies that it fails on errors.

Reply:
Done